ray-operator/test/migration/post-migration-check.sh

```
@@ -0,0 +1,476 @@
+        else
+            log_check "FAIL" "enableIngress should have been set to false"
+        fi
+    else
+        log_check "PASS" "enableIngress check not applicable"
+    fi
+    
+    echo ""
+    echo "--- Checking Head Group ---"
+    
+    # Check head group initContainers
+    local pre_head_init=$(echo "$pre_json" | jq -r '[.spec.headGroupSpec.template.spec.initContainers[].name] // []')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)